VideoWeb/VideoWeb/ClientApp/src/sass/vendor-modifications/_ngx-toastr-modifications.scss

```diff
@@ -0,0 +1,13 @@
+.toast-top-right {
+    top: 70px;
+    right: 20px;
+}
+
+.toast-top-center {
+    top: 70px !important;
+    @media only screen and (min-width: $breakpoint-medium) {
+        top: 70px !important;
```

Comment:
just a query, did we get this style from Neil?

Reply:
We imported the properties from the already toastr global styles we have in the code base